charts/gha-runner-scale-set-controller/templates/deployment.yaml

```
@@ -25,7 +25,7 @@ spec:
       labels:
         app.kubernetes.io/part-of: gha-rs-controller
         app.kubernetes.io/component: controller-manager
-        app.kubernetes.io/version: {{ .Chart.Version }}
+        app.kubernetes.io/version: {{ include "gha-runner-scale-set-controller.chart" . }}
```

```suggestion
        app.kubernetes.io/version: {{ .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
```
I really would love to keep the current versioning instead of applying new `gha-rs-` prefix on the version field. However, we can sanitize the chart version which should unblock your usecase. WDYT?

---

Either way, we just want it to work. I believe that was used because the include is used elsewhere

---

Indeed, I have no opinion about how the versioning should look like. Just want to have it working with Flux.